scenes/game/game.gd

```diff
@@ -354,6 +353,16 @@ func begin_remote_game(game_start_message):
 	starting_message = game_start_message.duplicate()
 	observer_mode = 'observer_mode' in game_start_message and game_start_message['observer_mode']
 	replay_button_visible = not observer_mode
+	print(game_start_message)
+	var starting_timer = game_start_message['starting_timer']*60 + 5
```

Comment:
when using magic numbers, let's have comments.
Maybe "starting_timer is in minutes, and add 5 seconds to buffer for UI loading"

Reply:
Added a comment and converted all times to seconds